src/System.ServiceModel.Syndication/tests/TestFeeds/atom_feeds.dat

```diff
@@ -119,14 +119,14 @@ multiple-categories.xml
 multiple-contributors-entry.xml
 multiple-contributors-source.xml
 multiple-contributors.xml
-###multiple-icons-source.xml   This operation is not supported for a relative URI.
-###multiple-icons.xml          This operation is not supported for a relative URI.
+multiple-icons-source.xml
+multiple-icons.xml
 multiple-related-matching.xml
 no-content-or-alternate.xml
 no-content-or-summary.xml
 no-name.xml
 prefixed-namespace.xml
-###rdf-extensions.xml    The method or operation is not implemented.
+###rdf-extensions.xml    Generated result file is different.
```

Comment:
Do we want to open an issue for this?

Reply:
This case can pass now. And the result is same with full framework's, I have removed the comments.
The `link-extensions.xml` file can also pass.
So these are not issue now.